internal/platform/activepodresources/supportversion.go

```diff
@@ -0,0 +1,104 @@
+package activepodresources
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/k8stopologyawareschedwg/deployer/pkg/deployer/platform"
+)
+
+// This file processes the known OCP platform versions, so far nightly, konflux-nightly, CI, dev-preview, and RC
+
+var (
+	// first build is always the nightly. Yet that doesn't necessarily mean that builds of other lanes
+	// (e.i CI, dev-preview) that produce the build after the specific nightly build date will certainly
+	// have the fix, thus we need to track them separately
+	StableSupportSince           = "4.20.999"
```

Comment:
we need to make logic on OCP versions, but this seems like a NROP version. Is this right? Ditto for all but `NightlySupportSince`. Is this what you meant in https://github.com/openshift-kni/numaresources-operator/pull/1748/commits/54a5afe71095cdc6dea855371cebac391f5e6923#r2253978141 ?

Reply:
yes, so these constants should be openshift/kubelet versions, and they don't use the `.999` versioning like NROP does